include/grpc/support/thd.h

```diff
@@ -66,9 +65,25 @@ int gpr_thd_new(gpr_thd_id *t, void (*thd_body)(void *arg), void *arg,
 /* Return a gpr_thd_options struct with all fields set to defaults. */
 gpr_thd_options gpr_thd_options_default(void);
 
+/* Set the thread to become detached on startup - this is the default. */
+void gpr_thd_options_set_detached(gpr_thd_options *options);
```

Comment:
My only reason is that this has been the default so far, and I wasn't so much willing to go back and edit all of the codebase that currently uses threads in a detached manner :-) But I can do that if that's desirable.